front-end/src/app/shared/models/transaction-groups/double-transaction-group.model.ts

```diff
@@ -10,6 +10,10 @@ export abstract class DoubleTransactionGroup {
 
   abstract hasEmployerInput(): boolean;
   abstract childHasEmployerInput(): boolean;
+  abstract hasParentCandidateInformationInput(): boolean;
+  abstract hasChildCandidateInformationInput(): boolean;
+  abstract hasParentElectionInformationInput(): boolean;
+  abstract hasChildElectionInformationInput(): boolean;
```

Comment:
We should consider this for a refactor.  Ideally, this would all be determined from examining the provided templateMap/childTemplateMap, but it would be great to at least get rid of having to implement the methods in every single DoubleTransactionGroup